testing/test_assertrewrite.py

```diff
@@ -608,6 +609,21 @@ def __repr__(self):
 
         assert r"where 1 = \n{ \n~ \n}.a" in util._format_lines([getmsg(f)])[0]
 
+    def test_custom_repr_non_ascii(self):
```

Comment:
Is this test something we want to keep around after dropping Python 2? If not, I suggest to leave a comment so we can remove the test itself when the time comes (the code itself already contains a "python2.x" so I think that's enough). 👍

Reply:
The comment in the code is enough to remove the code, but the test doesn't have any reference so I think it will mostly stay around. One alternative is to skip the test on python 3,this makes it clear it is python2 only.

Reply:
cool, yeah I added a comment to the test